packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx

```diff
@@ -2,31 +2,39 @@ import React from 'react';
 import { fireEvent, render, screen } from '@testing-library/react';
 import { CFDAccountCard } from '../cfd-account-card';
 import { TCFDAccountCard } from '../props.types';
+import { mockStore } from '@deriv/stores';
+import CFDProviders from '../../cfd-providers';
 
 type TMockPlatformAccounts = {
-    account_type: string;
+    account_type?: 'demo' | 'real';
     balance: number;
-    platform: string;
+    platform?: string;
     display_balance: string;
     display_login: string;
-    landing_company_short: string;
+    landing_company_short?: 'svg' | 'labuan' | 'bvi' | 'malta' | 'maltainvest' | 'vanuatu' | 'seychelles';
```

Comment:
We do not support 'seychelles', can we remove it from here or if it is being used in any other place.
If coming from the API types then it needs to be updated in the BE code as well.

Reply:
Done